snscrape/modules/twitter.py

```diff
@@ -317,6 +323,34 @@ def _tweet_to_tweet(self, tweet, obj):
 		kwargs['conversationId'] = tweet['conversation_id'] if 'conversation_id' in tweet else int(tweet['conversation_id_str'])
 		kwargs['lang'] = tweet['lang']
 		kwargs['source'] = tweet['source']
+                
+		# https://developer.twitter.com/en/docs/tutorials/filtering-tweets-by-location								     
+
+		coordinates = None						     
+		if tweet['coordinates']:
+			_coords = tweet['coordinates']['coordinates']
+			# coordinates root key (if present) presents coordinates in the form [LONGITUDE, LATITUDE]
+			if _coords and len(_coords) == 2:							     
+				coordinates = Coordinates(_coords[0], _coords[1])
+		elif tweet['geo']:
+			_coords = tweet['geo']['coordinates']
+			# coordinates root key (if present) presents coordinates in the form [LATITUDE, LONGITUDE]							     							     
+			if _coords and len(_coords) == 2:							     
+				coordinates = Coordinates(_coords[1], _coords[0])
+
+		
+		if tweet['place']:
+			kwargs['place'] = tweet['place']['full_name'] # take the place full name
```

Comment:
```suggestion
			kwargs['place'] = tweet['place']['full_name']
```

Redundant comment, the code already makes this clear.